src/test/java/core/basesyntax/transaction/FruitTransactionTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.transaction;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+    @Test
+    void correctReturnFor_balanceOperation() {
+        FruitTransaction.Operation b = FruitTransaction.Operation.fromCode("b");

```

Comment:
Do you really need to test that Java and method equals in fromCode() works well? 
I don't think that this test is reasonable